src/gbd_mapping/cause_template.py

```diff
@@ -4,7 +4,7 @@
 
 Any manual changes will be lost.
 """
-from typing import Tuple, Union
+from __future__ import annotations
```

Comment:
We still need these even when dropping 3.9 support? I can never remember when we need this...

Reply:
I think its for all the pandas series and indices

Reply:
Nevermind I am wrong and don't remember either.